api/types/mount/mount.go

```diff
@@ -56,3 +60,12 @@ type Driver struct {
 	Name    string            `json:",omitempty"`
 	Options map[string]string `json:",omitempty"`
 }
+
+// TmpfsOptions defines options specific to mounts of type "tmpfs".
+type TmpfsOptions struct {
+	// RawOptions is the raw string passed to mount(2).
+	// e.g. "rw,noexec,nosuid,size=65536k"
+	// RawOptions can contain "ro" or "rw" but needs to be consistent with
+	// Mount.ReadOnly
+	RawOptions string `json:",omitempty"`
```

Comment:
I think swarmkit is using `Mode` here.

```go
type Mount_TmpfsOptions struct {
	// Size sets the size of the tmpfs, in bytes.
	//
	// This will be converted to an operating system specific value
	// depending on the host. For example, on linux, it will be convered to
	// use a 'k', 'm' or 'g' syntax. BSD, though not widely supported with
	// docker, uses a straight byte value.
	//
	// Percentages are not supported.
	SizeBytes int64 `protobuf:"varint,1,opt,name=size_bytes,json=sizeBytes,proto3" json:"size_bytes,omitempty"`
	// Mode of the tmpfs upon creation
	Mode os.FileMode `protobuf:"varint,2,opt,name=mode,proto3,customtype=os.FileMode" json:"mode"`
}
```